web/src/cypress/support/profile.ts

```diff
@@ -135,75 +142,94 @@ function addNotificationRule(
       .then(prof => addNotificationRule({ ...nr, userID: prof.id }))
   }
-  if (!nr.cmID) {
+
+  if (!nr.contactMethodID) {
     return cy
-      .addContactMethod({ ...nr.cm, userID: nr.userID })
-      .then(cm => addNotificationRule({ ...nr, cmID: cm.id }))
+      .addContactMethod({ ...nr.contactMethod, userID: nr.userID })
+      .then(cm => addNotificationRule({ ...nr, contactMethodID: cm.id }))
   }
 
-  const query = `mutation addNR($input: CreateNotificationRuleInput!){
-        createNotificationRule(input: $input) {
-            id
-            delay: delay_minutes
-            cmID: contact_method_id
-            cm: contact_method {
-                id
-                name
-                type
-                value
-            }
+  const mutation = `
+    mutation ($input: CreateUserNotificationRuleInput!) {
+      createUserNotificationRule(input: $input) {
+        id
+        delayMinutes
+        contactMethodID
+        contactMethod {
+          id
+          name
+          type
+          value
         }
-    }`
+      }
+    }
+  `
 
   return cy
-    .graphql(query, {
+    .graphql2(mutation, {
       input: {
-        user_id: nr.userID,
-        delay_minutes: nr.delay || c.integer({ min: 0, max: 15 }),
-        contact_method_id: nr.cmID,
+        userID: nr.userID,
+        contactMethodID: nr.contactMethodID,
+        delayMinutes: nr.delayMinutes || c.integer({ min: 0, max: 15 }),
       },
     })
-    .then(newNR => {
-      newNR = newNR.createNotificationRule
+    .then(res => {
+      res = res.createUserNotificationRule
+
       const userID = nr && nr.userID
-      newNR.userID = userID
-      newNR.cm.userID = userID
-      return newNR
+      res.userID = userID
+      res.contactMethod.userID = userID
+
+      return res
     })
 }
+
 function clearContactMethods(id: string): Cypress.Chainable {
-  const list = `{
-        user(id: "${id}") {
-            contact_methods { id }
+  const query = `
+    query($id: ID!) {
+      user(id: $id) {
+        contactMethods {
+          id
         }
-    }`
-  return cy.graphql(list).then(res => {
-    if (!res.user.contact_methods.length) return
-    res.user.contact_methods.forEach((cm: any) => {
-      cy.graphql(`
-                mutation{
-                    deleteContactMethod(input:{id:"${cm.id}"}) {deleted_id}
-                }
-            `)
+      }
+    }
+  `
+
+  const mutation = `
+    mutation($input: [TargetInput!]!) {
+      deleteAll(input: $input)
+    }
+  `
+
+  return cy.graphql2(query, { id }).then(res => {
+    if (!res.user.contactMethods.length) return
+
+    res.user.contactMethods.forEach((cm: any) => {
+      cy.graphql2(mutation, {
+        input: [
+          {
+            type: 'contactMethod',
+            id: cm.id,
+          },
+        ],
+      })
     })
   })
 }
-function resetProfile(prof?: Profile): Cypress.Chainable<Profile> {
+
+function resetProfile(prof?: Profile): Cypress.Chainable<boolean> {
```

Comment:
Since this should always be true, or it would error/fail, maybe just return `Cypress.Chainable` like we do with `clearContactMethods` so it's not tied to the mutation at all.